registry/storage/driver/s3-aws/s3.go

```diff
@@ -549,9 +549,9 @@ func (d *driver) Reader(ctx context.Context, path string, offset int64) (io.Read
 
 // Writer returns a FileWriter which will store the content written to it
 // at the location designated by "path" after the call to Commit.
-func (d *driver) Writer(ctx context.Context, path string, append bool) (storagedriver.FileWriter, error) {
```

Comment:
Setting a variable with the name `append` overwrites the built-in function which meant I couldn't use it further down.